Documentation/installation/alibabacloud-eni.rst

```diff
@@ -162,12 +134,11 @@ connect to the AlibabaCloud API.
     $ kubectl create -f cilium-secret.yaml
 
 
-Deploy Cilium
-=============
+**Install Cilium:**
 
-.. include:: k8s-install-download-release.rst
+.. include:: ../installation/k8s-install-download-release.rst
```

Comment:
This will break if we ever include this from a file in a different level of hierarchy. Could you please try with an absolute path instead?

```suggestion
.. include:: /installation/k8s-install-download-release.rst
```

Reply:
Done